source/persistency/PetaloPersistencyManager.cc

```diff
@@ -285,18 +284,18 @@ void PetaloPersistencyManager::StoreIonizationHits(G4VHitsCollection* hc)
 
 void PetaloPersistencyManager::StoreSensorHits(G4VHitsCollection* hc)
 {
-  std::map<G4int, SensorHit*> mapOfHits;
+  std::map<G4int, PetSensorHit*> mapOfHits;
 
-  SensorHitsCollection* hits = dynamic_cast<SensorHitsCollection*>(hc);
+  PetSensorHitsCollection* hits = dynamic_cast<PetSensorHitsCollection*>(hc);
   if (!hits) return;
 
   std::vector<G4int > sensor_ids;
   for (size_t i=0; i<hits->entries(); i++) {
 
-    SensorHit* hit = dynamic_cast<SensorHit*>(hits->GetHit(i));
+    PetSensorHit* hit = dynamic_cast<PetSensorHit*>(hits->GetHit(i));
     if (!hit) continue;
 
-    int s_id  = hit->GetPmtID();
+    int s_id  = hit->GetSnsID();
```

Comment:
Why do we need to change from `GetPmtID` to `GetSnsID`? Why does `GetPmtID` remain in `PetSensorsEventAction.c?

Reply:
> Why do we need to change from GetPmtID to GetSnsID? 

It's just a name: it is clearer if we talk about _sensors_ instead of _pmts_, since we don't use PMTs at all in the system.

> Why does GetPmtID remain in `PetSensorsEventAction.c?

That's a mistake, I'll fix it. We should add a test for that actions, by the way.

Reply:
Yes, I think we should make a test for the new TOF output files.

Reply:
Actually I was referring to a test that uses `PetSensorEventAction`...

Reply:
Yes, sorry, I referred to the other one but I saw what you meant.